### ISSUE TITLE
Parse Thousands ("1.1k") in Social Metrics

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/github-scraper#108]
Currently the stats for a repo don't work beyond a thousand:
https://github.com/nelsonic/github-scraper/blob/1d943809bb0565f1c84d416949d1c59ea7c58ebc/lib/repo.js#L23-L28

This is a _really_ good problem to have. e.g: https://github.com/dwyl/start-here
<img width="1400" alt="dwyl-start-here-1 2k" src="https://user-images.githubusercontent.com/194400/77957567-97dd1f00-72cb-11ea-88bc-456c6b267402.png">

**Comment by SimonLab** [comment]
I had a quick look at how to update the regex but I think it might be complicated to do it in just one.
I'm thinking to:
- first replace the string which terminates by a dot then a number and "k", for example 3.2k. 
In this case we want to add "00" at the end of the string
- if the string doesn't contain a "." we then try to match a string finishing by just "k", e.g. 3k. In this case we want to replace the "k" by "000"
- finally we make sure that we replace non numbers by empty string

```js
var stringNumber = "4.3k";
stringNumber
.replace(/\.(\d)k$/, "$100") // $1 match the digit \d
.replace(/k$/, "000")
.replace(/[^0-9]/g, '')
```

I think this should work however I'm not sure this is the best or nicest solution. Moreover this will work for just "k", are there any repositories with more that a million stars "M"?

**Comment by nelsonic** [comment]
@SimonLab thanks for these lines! 👨‍💻 

As far as I'm aware, the most popular repo on GitHub is currently **310k**: https://github.com/freeCodeCamp/freeCodeCamp
![most-pop-repo](https://user-images.githubusercontent.com/194400/78450391-3247b480-7676-11ea-88f6-8b02cd19bd5a.png)

Not expecting them to reach 1M any time soon. ⏳ 
But ...  I've included a couple of test cases and lines in the `parse_int/1` function to handle the case. 👍

**Comment by nelsonic** [comment]
This is included in PR #109 🚀
